sycl/include/sycl/ext/intel/experimental/esimd/math.hpp

```diff
@@ -1723,6 +1723,13 @@ __ESIMD_API std::enable_if_t<__ESIMD_DNS::is_esimd_scalar<T>::value &&
   return __ESIMD_NS::bfn<FuncControl>(src0, src1, src2);
 }
 
+/// rdtsc - get the value of rdtsc register.
```

Comment:
rdtsc stands for ReaD TimeStamp Counter (I think). Can you please make it more clear in the comment here?

Reply:
GenX returns 4 integers (which is probably the content of the drtsc register (if it is a register).
The function only returns uint64 (part of what GenX can return), please don't use the word register in this decription.